src/InstrumentationEngine/ProfilerManager.h

```diff
@@ -200,12 +200,9 @@ namespace MicrosoftInstrumentationEngine
 
         HRESULT CreateNewMethodInfo(_In_ FunctionID functionId, _Out_ CMethodInfo** ppMethodInfo);
 
-        CLogging* GetLogging();
```

Comment:
I assume this didn't have an implementation.

Reply:
Correct.